www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/reverse_proxy.volt

```diff
@@ -257,7 +257,7 @@
                             <th data-column-id="FromPort" data-type="string">Port</th>
                             <th data-column-id="accesslist" data-type="string" data-visible="false">Access List</th>
                             <th data-column-id="basicauth" data-type="string" data-visible="false">Basic Auth</th>
-                            <th data-column-id="DnsChallenge" data-type="boolean" data-formatter="boolean" data-visible="false">DNS-01</th>
+                            <th data-column-id="DnsChallenge" data-type="boolean" data-formatter="boolean" data-visible="false">DNS-01 challenge</th>
```

Comment:
all these are not properly translated BTW

Reply:
Thank you for the feedback, I fixed all of that in a new PR. 😊